app/frontend/src/styles/application/jobseekers/search-header.scss

```diff
@@ -0,0 +1,33 @@
+.search-results__header {
+  display: flex;
+  flex-direction: column;
+
+  @include govuk-media-query($from: tablet) {
+    flex-direction: row;
+  }
+
+  &-stats {
```

Comment:
That's an interesting language feature I wasn't aware of (joining a class name with `&` and `-`).

Reply:
can explain exactly what it does if you want - the &. the - is not a feature its part of the class name